src/main/resources/graphql/service/mutation.graphqls

```diff
@@ -0,0 +1,152 @@
+type Mutation {
+    """
+    Create a quiz.
+    """
+    createQuiz(input: CreateQuizInput!): Quiz!
+
+    """
+    Modify a quiz.
+    """
+    mutateQuiz(assessmentId: UUID!): QuizMutation!
+
+    """
+    Delete a quiz.
+    """
+    deleteQuiz(assessmentId: UUID!): UUID!
```

Comment:
Can this be moved into the QuizMutation type? This'll make it easier later to implement authorization

Reply:
I think thematically it would make more sense if it would be outside of the QuizMutation

Reply:
This mutation should not be accessible from the outside at all I think. I can mark it as deprecated. Deletion should happen when the content is deleted